packages/terra-action-footer/docs/README.md

```diff
@@ -0,0 +1,45 @@
+# Terra Action Footer
+
+The terra-action-footer component is a footer bar that contains two sockets, start and end, to place items in the form of buttons and hyperlinks used to perform actions. If no actions are provided, the footer bar defaults to a 10px equivalent height and maintains the top border.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-action-footer`
+
+## Usage
+
+### Action Footer with a Start and End Actions
```

Comment:
Can you rephrase this to "Action Footer with a Start and End Action" or "Action Footer with Start and End Actions"?